pipelines/deployment.yml

```diff
@@ -966,6 +966,9 @@ jobs:
         REDIS_SERVICE: ras-redis
         UAA_CLIENT_ID: 'response_operations'
         UAA_CLIENT_SECRET: ((ci_response_operations_client_secret))
+        RAS_NOTIFY_CONFIRM_PASSWORD_CHANGE_TEMPLATE: 83bbcdeb-8152-4d48-9cf2-5010f0db0ee0
```

Comment:
Same as in the other PR I would remove the RAS prefix to the variables as we have been moving away from this.

Reply:
As with other PR, we need to decide between refactoring and consistency :)